options/license/Noweb

```diff
@@ -6,4 +6,4 @@ Noweb is available free for any use in any field of endeavor. You may redistribu
 
 You may sell noweb if you wish. For example, you may sell a CD-ROM including noweb.
 
-You may sell a derived work, provided that all source code for your derived work is available, at no additional charge, to anyone who buys your derived work in any form. You must give permisson for said source code to be used and modified under the terms of this license. You must state clearly that your work uses or is based on noweb and that noweb is available free of change. You must also request that bug reports on your work be reported to you.
+You may sell a derived work, provided that all source code for your derived work is available, at no additional charge, to anyone who buys your derived work in any form. You must give permission for said source code to be used and modified under the terms of this license. You must state clearly that your work uses or is based on noweb and that noweb is available free of change. You must also request that bug reports on your work be reported to you.
```

Comment:
A typo in a LICENSE :laughing:

Reply:
Upstream certainly has the typo too: https://fedoraproject.org/wiki/Licensing:Noweb?rd=Licensing/Noweb, not sure if this is the defintive source.

Reply:
Does anyone want to make a patch and submit it upstream, or shall I do it?

Reply:
@luzpaz please try to patch it upstream - and revert the change in our copy, as it will be overwritten anyway (by our tool chain)

Reply:
Then we might not need to wait for the upstream, and merge other fixes in this PR first.